### ISSUE TITLE
Request - Improve organization of Arctos log-in page

### ISSUE THREAD

**Issue by zygoballus** [ArctosDB/arctos#8971]
Help us understand your request (check below):

- [ ] search or data request (help with SQL or provide the specific fields you are trying to retrieve for your collection)
- [ ] add a new Github user to the Arctos Users team
- [ ] a new barcode series
- [ ] a new term for data entry or management 
- [ ] a change or new report or label
- [ ] update to an Agent (split, merge, or other type of change)
- [ ] bulkload changes (to Agents, Identifiers, etc that are not already available as a bulkloading tool)
- [x] new feature or otherwise a good idea to make Arctos even better
- [ ] quarantine taxon name and suggested replacement (please supply links)
- [ ] API key request (see https://handbook.arctosdb.org/documentation/api.html for requirements)
- [ ] error or bug (please include enough specific information to reproduce)
- [ ] general inquiry (or email arctoscontact@googlegroups.com)
- [ ] other

**Describe what you're trying to do** 
The log-in page is the second most important UI in Arctos (after the search page). However, it's layout and organization are confusing and not optimized for common use cases. Here are some suggestions for improving it:

- The 'no account' log-in should be the first option presented rather than being hidden at the bottom of the page.
- The blurb about using the Arctos API should be moved to the bottom instead of being at the top as this is not a common use case.
- The link to recover your password should be immediately below the Local Account login.
- The interfaces for logging in with TACC, Google, and GitHub are visually confusing (squares within squares that are redundantly labeled). Just use a button for each (that looks like a button).

**Comment by mkoo** [comment]
agreed-- on tech agenda already. thx!
